### PR TITLE
[vim-scripts/cpp.vim] Add highlight support for QT

### PR DESCRIPTION
**Pull request by shjinyuan**

### CODE REVIEW

File: .vim/sky8336/plugin.vim [hunk 1]
```diff
@@ -90,6 +92,11 @@ let plugin_enalbe_rainbow_parentheses = 0
 	if plugin_enalbe_rainbow_parentheses == 1
 		Plug 'kien/rainbow_parentheses.vim'
 	endif
+	
+	if plugin_enalbe_Qt_highlight_support == 1
```
Comment (sky8336) [inline .vim/sky8336/plugin.vim]: 1. please modify the spelling error: enalbe -> enable

**Comment by shjinyuan** [comment]
correct the spell issue

File: .vim/sky8336/plugin.vim [hunk 2]
```diff
@@ -89,11 +89,11 @@ let plugin_enalbe_Qt_highlight_support = 1
 		Plug 'octol/vim-cpp-enhanced-highlight'
 	endif
 
-	if plugin_enalbe_rainbow_parentheses == 1
+	if plugin_enable_rainbow_parentheses == 1
```
Comment (sky8336) [inline .vim/sky8336/plugin.vim:93]: modify the spelling error : 
1.
36  let plugin_enalbe_rainbow_parentheses = 0       
...

2.
plugin_setting.vim:
681 if plugin_enalbe_rainbow_parentheses == 1

**Review by sky8336 — COMMENTED** [review]
please modify the spelling error again

**Comment by shjinyuan** [comment]
correct spell issue for plugin_enable_Qt_highlight_support

**Pull request merged**